lib/modules/deployment/contract_deployer.js

```diff
@@ -44,7 +44,13 @@ class ContractDeployer {
       }
       let contractName = arg.substr(1);
       self.events.request('contracts:contract', contractName, (referedContract) => {
-        cb(null, referedContract.deployedAddress);
+        if(referedContract.deployedAddress !== undefined) {
```

Comment:
can it just be `cb(null, (referedContract.deployedAddress || '0x0000000000000000000000000000000000000000'));`

Reply:
Absolutely